docs/quickstart-aws.md

```diff
@@ -189,7 +189,7 @@ As worker nodes are managed by machine-controller, they can be scaled up and dow
 (including to 0) using Kubernetes API.
 
 ```bash
-kubectl --namespace kube-system scale machinedeployment/pool1-deployment --replicas=3
+kubectl --namespace kube-system scale machinedeployment/<CLUSTER-NAME>-pool1 --replicas=3
```

Comment:
In the case of AWS, there are now 3 MachineDeployments instead of 1 (each per AZ). Do you want to update the document to mention that or should we do it in a follow-up?

Reply:
When telling me a bit more about it I can do it.

Reply:
Those names are dynamic, 
here https://github.com/kubermatic/kubeone/blob/master/examples/terraform/aws/output.tf#L52
here https://github.com/kubermatic/kubeone/blob/master/examples/terraform/aws/output.tf#L83
here https://github.com/kubermatic/kubeone/blob/master/examples/terraform/aws/output.tf#L114

all 3 of them. As you can see they depend on AZ where MachineDeployment will be pointed to.

Reply:
Okay. Any good way how the user quickly can get aware of it w/o manually scan the generated files?

Reply:
I think the best way is to use `kubectl get machinedeployments -n kube-system`.

Reply:
yeah, generated names will be presented in `get machinedeployments` list